components/sup/src/manager/spec_watcher.rs

```diff
@@ -29,8 +30,8 @@ use manager::service::ServiceSpec;
 
 static LOGKEY: &'static str = "SW";
 const WATCHER_DELAY_MS: u64 = 2_000;
-const SPEC_FILE_EXT: &'static str = "spec.toml";
```

Comment:
I'd have argued against dropping the `.toml` extension here. The user in mind here is someone using a configuration management tool who would be writing these files when setting up a host (the same reasoning for having `.sh` on the Plan file). So in effect, this *could* be in someone's text editor. I know that made the Rust file stem logic extra crazy (as I see you found below) but not a big deal either way. Anyway, that was my thinking.